web/packages/teleport/src/services/webUiVersion/webUiVersion.ts

```diff
@@ -34,7 +34,7 @@ export function withUnsupportedLabelFeatureErrorConversion(
         'We could not complete your request. ' +
           'Your proxy may be behind the minimum required version ' +
           `(${getWebUiVersion()}) to support adding resource labels. ` +
-          'Remove labels and try again or upgrade your proxy version.'
+          'Upgrade your proxy version or remove labels and try again.'
```

Comment:
This error message should state something about upgrading "all" their proxies. They might think they have upgraded all the proxies already. Thats why they have this new Web UI served already. The problem here is if they are reaching an old _other_  proxy unexpectedly.

Reply:
it looks like you are on the outdated commit, the newest changes says `'Ensure all proxies are upgraded or remove labels and try again.'`